src/Groups/group_characters.jl

```diff
@@ -600,6 +626,14 @@ function Base.:^(chi::GAPGroupClassFunction, n::IntegerUnion)
     return GAPGroupClassFunction(chi.table, chi.values ^ n)
 end
 
+function Base.:^(chi::GAPGroupClassFunction, tbl::GAPGroupCharacterTable)
+    return GAPGroupClassFunction(tbl, GAP.Globals.InducedClassFunction(chi.values, tbl.GAPTable))
+end
+
+function complex_conjugate(chi::GAPGroupClassFunction)
```

Comment:
```suggestion
function conj(chi::GAPGroupClassFunction)
```
This is the name for complex conjugate so far. We should try to be consistent here. We can add the alias later.

Reply:
@thofma Thanks. I will change the name.
(However, I think that `conj` is a bad name. In particular in contexts where "algebraic conjugacy" occurs, and complex conjugation is just a special case.)

Reply:
Agreed, but it's a name already defined by `Julia`, so we are adding methods to it... but we certainly could have alias names.